ethcore/sync/Cargo.toml

```diff
@@ -4,26 +4,27 @@ name = "ethcore-sync"
 version = "1.12.0"
 license = "GPL-3.0"
 authors = ["Parity Technologies <admin@parity.io>"]
+edition = "2018"
 
 [lib]
 
 [dependencies]
 client-traits = { path = "../client-traits" }
-common-types = { path = "../types" }
+types = { package = "common-types", path = "../types" }
```

Comment:
nit: I for one prefer to use the full name for this one and update the imports. It's fine like this too, just saying. I do prefer what you did below with `light`/`network`/`devp2p` so yeah, I guess I'm inconsistent.
:)

Reply:
sure, sounds ok to me